.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### Summary
+
+_Describe main changes going into those pull requests._
+
+_Use "closes #(issue number)" to enable GitHub to close issue when PR is merged_
+
+### Areas Of Interest
+
+_Any specific places you want to bring attention of the reviewers? Some places you are uncertain in/some design decisions. (If none remove this section)_
+
+### Checklist
+
+#### Design
+
+- [ ] Web API layer contains as little logic as possible
+- [ ] Domain objects use semantic names
+
+#### Perfromance
+
+- [ ] In case of any DB query was changed: attach generated SQL for review
+- [ ] Number of DB roundtrips should be as low as possible
+
+#### Unit Tests
+
+- [ ] Naming: Initial state and expected result are properly stated
+- [ ] Test follows AAA pattern
```

Comment:
I'd suggest to add link to pattern description (e.g. https://www.typemock.com/unit-test-patterns-for-net#aaa)

Reply:
👍 I'll fix it